### DISCUSSION TITLE
[M-Z] (12 Settembre 2018) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#428] [Esami & Esoneri M-Z]
## Traccia

<img width="923" alt="image" src="https://github.com/user-attachments/assets/b9979b25-8870-49bd-8efc-81f857a49295">

**Comment by rimaout** [comment]
Una relazione è di equivalenza se soddisfa le proprietà della riflessività, simmetria e transitività.

- Esempio: $R$ = { $(1,1), (1,2), (1,3), (2,1), (2,2), (2,3), (3,1), (3,2), (3,3)$ }

Una classe di equivalenza di un determinato elemento $x$ è l'insieme di elementi che sono in relazione con $x$.

- Nel caso di $R$ dell'esempio, abbiamo che tutti gli elementi sono in relazione tra loro, quindi $[1] = [2] = [3] =$ { $1,2,3$ }

L'insieme quoziente è composto da tutte le classi di equivalenza della relazione.

- In questo esempio, l'insieme quoziente di $R$ è $Q =$ { { $1,2,3$ } }

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180912.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180912.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/dd01db35-be56-40f0-ad01-61ad07849b39)
